ComputerVision/SDK/AnalyzeImage/Program.cs

```diff
@@ -11,24 +11,22 @@ namespace ImageAnalyze
     class Program
     {
         private const string subscriptionKey = "0123456789abcdef0123456789ABCDEF";  //change this with your subscription key
-        private const string remoteImageUrl =
-            "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample3.png";
+        private const string remoteImageUrl = "https://github.com/harishkrishnav/cognitive-services-dotnet-sdk-samples/raw/master/ComputerVision/Images/sample3.png";
 
         // Specify the features to return
         private static readonly List<VisualFeatureTypes> features =
             new List<VisualFeatureTypes>()
-        {
+            {
             VisualFeatureTypes.Categories, VisualFeatureTypes.Description,
             VisualFeatureTypes.Faces, VisualFeatureTypes.ImageType,
             VisualFeatureTypes.Tags, VisualFeatureTypes.Adult,
             VisualFeatureTypes.Color, VisualFeatureTypes.Brands,
             VisualFeatureTypes.Objects
-        };
+            };
 
         static void Main(string[] args)
         {
-            ComputerVisionClient computerVision = new ComputerVisionClient(
-                new ApiKeyServiceClientCredentials(subscriptionKey),
+            ComputerVisionClient computerVision = new ComputerVisionClient(new ApiKeyServiceClientCredentials(subscriptionKey),
```

Comment:
You can skip passing the DelegatingHandler to the constructor, and just pass the key credentials.